packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # prosekit
 
+## 0.22.0-beta.18
+
+### Patch Changes
+
+- [`1ed0635`](https://github.com/ocavue/prosekit/commit/1ed063562a03866e4f6427ffdd30fcf421310460) ![](https://prosekit.dev/b/extensions)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)